### ISSUE TITLE
BUG: BottomToUp arrow icon overlapped by ChatBot Icon

### ISSUE THREAD

**Issue by Amruta7203** [Nactore-Org/Nacto-Care#170]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What went down?

The BottomToUp arrow icon, is being obscured by the ChatBot icon. This can lead to a poor user experience as the functionality of the arrow icon might be hidden or confusing.

Impact:
Users might miss the functionality of the BottomToUp arrow icon.
The overall design of the webpage might appear visually cluttered or confusing.

This can be resolved by adjusting the position of either of the icons above or below the other, such that they are aligned vertically on y-axis.

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by Harshil-Jani** [comment]
@Amruta7203 If you are working on this then please comment below else we can assign under #191

**Comment by Amruta7203** [comment]
yes @Harshil-Jani I am working on it and would like to collaborate with #191. So that we can raise the PR together.
@mukulpythondev

**Comment by Harshil-Jani** [comment]
@Amruta7203 I don't think it's big of an issue to have a collaboration over here and eventually it will lead to merge conflicts for tiny things. Since you are involved with other contributions let's give @mukulpythondev a chance so that we can speed up things a bit.

**Comment by mukulpythondev** [comment]
@Harshil-Jani Thanka for giving me this opportunity i will raise the pr till tonight.

**Comment by MastanSayyad** [comment]
Hey, @Harshil-Jani I would like to take on this task as it is taking way more time to get fixed by the assignees, New contributors will continuously raise the issues regarding this overlapping so it should be fixed quickly, please assign it to me.

**Comment by Harshil-Jani** [comment]
@Amruta7203 Just acknowledge in case if you are still working to fix this issue. Just wanted to keep a check on the progress.

**Comment by ansh-h-mehta** [comment]
Hey @Harshil-Jani  is there any update regarding it? If not can it be assigned to me?

**Comment by Harshil-Jani** [comment]
Sure @ansh-h-mehta

**Comment by ansh-h-mehta** [comment]
Hey @Harshil-Jani Thank you for assigning the bug.
